src/app/api/types/EventsRequestType.ts

```diff
@@ -0,0 +1,19 @@
+import { AxiosResponse } from 'axios';
```

Comment:
Can we put all the events types in one file?

Reply:
Would it be better if there was an events types folder?

Reply:
Perhaps! Basically a little better organization. The types themselves seemed small enough to warrant compiling into one file, but if you think a folder would be better with types modularized into their own separate files as a consistent practice, then we can do that 👍